images/kafka/config/main.tf

```diff
@@ -9,11 +9,20 @@ variable "extra_packages" {
   default     = ["kafka", "openjdk-11-default-jvm"]
 }
 
+variable "environment" {
+  default = {}
+}
+
 data "apko_config" "this" {
   config_contents = file("${path.module}/template.apko.yaml")
   extra_packages  = var.extra_packages
 }
 
 output "config" {
-  value = jsonencode(data.apko_config.this.config)
+  value = jsonencode(merge(
+    {
+      environment = var.environment
```

Comment:
DNM, something's off here. Adding to env vars in the apko.yaml works however this doesn't. Needs work on my end

Reply:
This has been resolved :)